src/components/Note/NoteAuthorList/NoteAuthorList.scss

```diff
@@ -13,7 +13,11 @@ $max-name-length: 112px;
   display: flex;
   flex-direction: row;
   align-items: center;
-  gap: $stack-author-rest-author-gap;
+
+  // If there's at least one additional author in the stack, apply the gap
+  &:has(> .note-rest-authors__container > *) {
```

Comment:
LGTM, could maybe change the `*` to a more specific selector (`figure`) but it probably doesn't matter that much

Reply:
I've used the start because I didn't really care about the type of the children, as long as some exist